osf/models/node.py

```diff
@@ -735,6 +723,20 @@ def comment_level(self, value):
     def get_absolute_url(self):
         return self.absolute_api_v2_url
 
+    @property
+    def permissions(self):
```

Comment:
We shouldn't need this anymore: we already have `AbstractNode#has_permission`, `AbstractNode#get_permissions`, and `AbstractNode#is_contributor`. Was this added to make checks for `user in node.permissions` work? If so, can you change those to use `node.is_contributor(user)` instead?